TOC.md

```diff
@@ -690,6 +690,7 @@
       - [`FLUSH PRIVILEGES`](/sql-statements/sql-statement-flush-privileges.md)
       - [`FLUSH STATUS`](/sql-statements/sql-statement-flush-status.md)
       - [`FLUSH TABLES`](/sql-statements/sql-statement-flush-tables.md)
+      - [`FOREIGN KEY`](/sql-statements/sql-statement-foreign-key.md)
```

Comment:
This looks wrong to me. I don't think there is a `FOREIGN KEY` statement, I think Foreign Keys are part of `CREATE TABLE` statements etc isn't it?

Reply:
Great, Maybe use `FOREIGN KEY Constraints` is better, just like the MySQL name. https://dev.mysql.com/doc/refman/8.0/en/create-table-foreign-keys.html

Reply:
Yes, they document foreign key constraints as part of the `CREATE TABLE` statement.